### ISSUE TITLE
Error: "Could not start Telegram Desktop!"

### ISSUE THREAD

**Issue by FTABoy** [telegramdesktop/tdesktop#4929]
version: 1.3.9 Alpha
steps: quit telegram - open again. following error appears.
![untitled](https://user-images.githubusercontent.com/25687276/42133952-b5f05154-7d47-11e8-9e8f-357e367be550.jpg)
(consider i'm using mtproto proxy)
logs:
[log.zip](https://github.com/telegramdesktop/tdesktop/files/2152759/log.zip)
thanks

**Comment by stek29** [comment]
Maybe AppData permissions are messed up?

**Comment by FTABoy** [comment]
@stek29 nope no permission has changed ... i think old version were fine and there is something wrong with 1.3.9?
here is an screenshot of AppData permissions:
![untitled](https://user-images.githubusercontent.com/25687276/42215008-6e1dca0a-7ed2-11e8-9a14-59db104068a6.jpg)

**Comment by Aokromes** [comment]
#4756 ?

**Comment by Aokromes** [comment]
@FTABoy this bug is still valid on 1.8.2?

**Comment by Aokromes** [comment]
no answer after months, closing.

**Comment by FTABoy** [comment]
@Aokromes  sorry for reopening this and late answer. 
yes i still have this issue. it happens when i boot my windows and if i open telegram fast while its still starting windows things up... i click on telegram icon it wont open. i click again several more time then it finally opens but show this me this:
![image](https://user-images.githubusercontent.com/25687276/73811382-95d87180-47ee-11ea-836d-acfa29d6cd78.png)
[2020.02.05 08:06:16] Executable path before check: C:/Users/Navid/AppData/Roaming/Telegram Desktop/Telegram.exe
[2020.02.05 08:06:16] FATAL: Could not move logging to 'C:/Users/Navid/AppData/Roaming/Telegram Desktop/log.txt'!
[log.zip](https://github.com/telegramdesktop/tdesktop/files/4157415/log.zip)
thanks.

**Comment by Aokromes** [comment]
can you try deleting the directory Telegram Desktop and install again? it looks really like stek said permissions are messed.

**Comment by jeffythedragonslayer** [comment]
This issue also affects me intermittently.

**Comment by EMMUR3** [comment]
I have the same issue right now. 

Yesterday I tried reinstalling it, but the same issue appeared again (I guess it was installing into the same AppData\Telegram Desktop folder and the old file was still there). But after restarting the pc it somehow worked... just to happen again next time I booted. I guess this error only appears when having "start with system" enabled in telegram.

The problem here is, that somethig blocks the log.txt file on startup, before telegram can access it. Trying to delete the file doesn't work either, since (even though I am the admin and only user on this PC) windows tells me that the file is being used by something. I already checked windows built-in recource monitor and the process explorer software to find any handle or dll accessing the file, but nothing. Even the "unlocker" tool (which usually can delete even used files), cant do a thing.

Did anyone find a solution or maybe even the programm/handle which is locking the file?

Cheers!
